core/src/main/java/com/microsoft/applicationinsights/internal/channel/samplingV2/TelemetryType.java

```diff
@@ -0,0 +1,19 @@
+package com.microsoft.applicationinsights.internal.channel.samplingV2;
+
+public enum TelemetryType {
```

Comment:
I am okay with this change. This is what you mentioned enables auto completion correct?

Reply:
Yes